steps/appgallery-deploy/1.5.1/step.yml

```diff
@@ -0,0 +1,140 @@
+title: Deploy to Huawei App Gallery
+summary: Uploads and releases apps to Huawei App Gallery
+description: |-
+  Uploads and releases apps to Huawei App Gallery.
+
+  This step is based on [AppGallery Connect API](https://developer.huawei.com/consumer/en/doc/development/AppGallery-connect-Guides/agcapi-getstarted)
+website: https://github.com/ferPrieto/steps-app-gallery-deploy
+source_code_url: https://github.com/ferPrieto/steps-app-gallery-deploy
+support_url: https://github.com/ferPrieto/steps-app-gallery-deploy/issues
+published_at: 2021-02-19T17:35:01.068025Z
+source:
+  git: https://github.com/ferPrieto/steps-app-gallery-deploy.git
+  commit: b09aaa21bf5f7d842a52bedad5815f21b97f2ab1
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- android
+- xamarin
+- react-native
+- flutter
+type_tags:
+- deploy
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: jq
+  apt_get:
+  - name: jq
+is_requires_admin_user: true
+is_always_run: true
+is_skippable: false
+run_if: ""
+inputs:
+- file_path: $BITRISE_APK_PATH
+  opts:
+    category: Config
+    description: Path to generated APK/AAB file to deploy.
+    is_expand: true
+    is_required: true
+    title: File path
+- huawei_filename: app_release_${BITRISE_BUILD_NUMBER}.apk
+  opts:
+    category: Config
+    description: Unique name of APK/AAB file upload.
+    is_expand: true
+    is_required: true
+    title: File name
+- huawei_app_id: null
+  opts:
+    category: Config
+    description: App ID can be found in the `App information` section in App Gallery
+      Connect.
+    is_expand: true
+    is_required: true
+    title: App ID
+- huawei_client_id: null
+  opts:
+    category: Config
+    description: Client ID generated in `AppGallery Connect -> Users and permissions
+      -> API key -> Connect API -> API client`
+    is_expand: true
+    is_required: true
+    title: Client ID
+- huawei_client_secret: null
+  opts:
+    category: Config
+    description: Key Secret generated in `AppGallery Connect -> Users and permissions
+      -> API key -> Connect API -> API client`
+    is_expand: true
+    is_required: true
+    is_sensitive: true
+    title: Key
+- opts:
+    category: Config
+    description: App release mode (Entire network | By phase)[Release Type](https://developer.huawei.com/consumer/en/doc/development/AppGallery-connect-References/agcapi-app-submit-with-file_v2)
+    is_expand: true
+    is_required: true
+    title: Release Type
+    value_options:
+    - Entire Network
+    - By Phase
+  release_type: Entire Network
+- opts:
+    category: Config
+    description: By default is marked as For Review (true). Setting to false will
+      submit the relase as a Draft.
+    is_expand: true
+    is_required: false
+    title: Submit for Review
+    value_options:
+    - true
+    - false
```

Comment:
Please make sure these are wrapped in double quotes (must be string)

Reply:
@tadam313 I've updated the PR, let me know if it's necessary to close this and create an entire new one. Cheers